assets/dev/js/editor/regions/panel/pages/elements/favorites/component.js

```diff
@@ -0,0 +1,80 @@
+import * as commands from './commands';
+import * as dataCommands from './commands-data';
+
+export default class Component extends $e.modules.ComponentBase {
+	/**
+	 * @inheritDoc
+	 */
+	__construct( args = {} ) {
+		super.__construct( args );
```

Comment:
`__construct` is an old approach created for use in ES5 modules. It's better to use `constructor` instead